### PR TITLE
Include `.js` extension on distributed resources

### PR DESCRIPTION
**Pull request by antonok-edm**
All of uBO's resources include the file extension on their resources as a convention. We should do the same for consistency.

No corresponding changes are required in our filter lists; `adblock-rust` already allows scriptlet injections to omit the `.js` extension.

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,78 +1,46 @@
...
-        "name": "brave-fix",
+        "name": "brave-fix.js",
         "aliases": [],
         "kind": {
             "mime": "application/javascript"
         },
         "resourcePath": "brave-fix.js"
     },
     {
-        "name": "brave-set-cookie-reload",
```
Comment (ShivanKaul) [inline metadata.json:27]: Are we deliberately removing these in this PR?

File: dist/resources.json [hunk 2]
```diff
@@ -1 +1 @@
...
...AmJiBza2lwOyBqKyspIHsKICAgICAgICAgIGlmIChyZW1vdmVkTm9kZXNbal0ubm9kZVR5cGUgPT09IDEpIHsKICAgICAgICAgICAgc2tpcCA9IGZhbHNlCiAgICAgICAgICAgIGJyZWFrCiAgICAgICAgICB9CiAgICAgICAgfQogICAgICB9CiAgICAgIGlmIChza2lwKSB7CiAgICAgICAgcmV0dXJuCiAgICAgIH0KICAgICAgdGltZXIgPSBzZWxmLnJlcXVlc3RJZGxlQ2FsbGJhY2socm1hdHRyLCB7IHRpbWVvdXQ6IDE3IH0pCiAgICB9CgogICAgY29uc3Qgc3RhcnQgPSAoKSA9PiB7CiAgICAgIHJtYXR0cigpCiAgICAgIGNvbnN0IG9ic2VydmVyID0gbmV3IE11dGF0aW9uT2JzZXJ2ZXIobXV0YXRpb25IYW5kbGVyKQogICAgICBvYnNlcnZlci5vYnNlcnZlKGRvY3VtZW50LCB7CiAgICAgICAgYXR0cmlidXRlczogdHJ1ZSwKICAgICAgICBhdHRyaWJ1dGVGaWx0ZXI6IFthdHRyXSwKICAgICAgICBjaGlsZExpc3Q6IHRydWUsCiAgICAgICAgc3VidHJlZTogdHJ1ZQogICAgICB9KQogICAgfQoKICAgIHNlbGYuYWRkRXZlbnRMaXN0ZW5lcignRE9NQ29udGVudExvYWRlZCcsIHN0YXJ0LCB7IG9uY2U6IHRydWUgfSkKICB9Cn0pKCk7Cg=="},{"name":"touch-fix.js","aliases":[],"kind":{"mime":"application/javascript"},"content":"Ly8gUmVtb3ZlIHRvdWNoIGNoZWNrcyBvbiBzZWxlY3RlZCBtb2JpbGUgc2l0ZXMKZGVsZXRlIERvY3VtZW50LnByb3RvdHlwZS5jcmVhdGVUb3VjaApkZWxldGUgRG9jdW1lbnQucHJvdG90eXBlLm9udG91Y2hzdGFydApkZWxldGUgRG9jdW1lbnQucHJvdG90eXBlLm9udG91Y2hlbmQKZGVsZXRlIERvY3VtZW50LnByb3RvdHlwZS5vbnRvdWNoY2FuY2VsCmRlbGV0ZSBEb2N1bWVudC5wcm90b3R5cGUuVG91Y2hFdmVudAo="}]
```
Comment (ShivanKaul) [inline dist/resources.json:1]: Let's smoke test this by running through https://github.com/brave/adblock-resources/pull/121

File: metadata.json (hunk 1 above)
Comment (antonok-edm) [reply metadata.json:27]: That's leftover cleanup from https://github.com/brave/adblock-resources/pull/123. I thought we had CI setup to verify these are done, but... :shrug:?

**Review by ShivanKaul — APPROVED** [review]

**Review by pes10k — APPROVED** [review]
lgtm

File: dist/resources.json (hunk 2 above)
Comment (antonok-edm) [reply dist/resources.json:1]: seems alright by my testing

**Pull request merged**